pyproject.toml

```
@@ -1,165 +1,170 @@
 [build-system]
-requires = ["setuptools>=61.0", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
 
 [project]
 name = "papermill"
 dynamic = ["version"]
 description = "Parameterize and run Jupyter and nteract Notebooks"
 readme = "README.md"
-license = {text = "BSD"}
+license = {text = "BSD-3-Clause"}
+requires-python = ">=3.10"
 authors = [
-    {name = "nteract contributors", email = "nteract@googlegroups.com"}
+    { name = "nteract contributors", email = "nteract@googlegroups.com" },
+]
+keywords = [
+    "jupyter",
+    "mapreduce",
+    "notebook",
+    "nteract",
+    "pipeline",
 ]
-keywords = ["jupyter", "mapreduce", "nteract", "pipeline", "notebook"]
 classifiers = [
     "Intended Audience :: Developers",
-    "Intended Audience :: System Administrators",
     "Intended Audience :: Science/Research",
-    "License :: OSI Approved :: BSD License",
+    "Intended Audience :: System Administrators",
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
     "Programming Language :: Python :: 3.13",
 ]
-requires-python = ">=3.10"
 dependencies = [
+    "aiohttp >=3.9.0; python_version == \"3.12\"",
```

nit: I do recommend embedded single quotes as before, that's the way Hatch(ling) actually transforms:

https://github.com/pypa/hatch/blob/b0609b3cec34346eb50e8defc53444ed3f9653c8/backend/src/hatchling/metadata/utils.py#L37-L39

```suggestion
    "aiohttp >=3.9.0; python_version == '3.12'",
```